app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -180,12 +151,40 @@ private void onBasemapLoaded() {
         setOnStatusChangedListener(null);
     }
 
+    private void setExtentOverIsrael() {
+        Envelope israelEnvelope = new Envelope(
```

Comment:
You can make it simpler. This method can get Point as parameter.
Change the constants to one Point constants if possible.